docs/products/postgresql/howto/prevent-full-disk.rst

```diff
@@ -0,0 +1,58 @@
+Prevent PostgreSQL full disk issues
+===================================
+
+If your Aiven for PostgreSQL service runs out of disk space, the service will start malfunctioning and precluding the proper backups creation. 
```

Comment:
Hmm. Precluding doesn't feel quite right here - I think it would need to be "which will preclude".

I'd probably just say "will start malfunctioning and backup will not work." (or, if you prefer, "will not work properly", but I assume "will not work" is accurate enough?)

(I quite like the use of malfunctioning, though)